Assets/MRTK/Providers/OpenXR/Scripts/OpenXRSpatialAwarenessMeshObserver.cs

```diff
@@ -81,7 +81,9 @@ private bool SetMeshComputeSettings(SpatialAwarenessMeshDisplayOptions option, S
         {
             MeshComputeSettings settings = new MeshComputeSettings
             {
+#if !UNITY_OPENXR_1_4_0_OR_NEWER
                 MeshType = (option == SpatialAwarenessMeshDisplayOptions.Visible) ? MeshType.Visual : MeshType.Collider,
```

Comment:
Should we inform the user that this parameter is being ignored in some way (maybe somewhere higher up in the call stack)?

Reply:
I think it's okay. There are other places in MRTK that use this `SpatialAwarenessMeshDisplayOptions` setting, so it's not completely ignored, and OpenXR was the only platform that claimed to support distinct mesh profiles from the runtime.

Reply:
> OpenXR was the only platform that claimed to support distinct mesh profiles from the runtime.
Which is actually untrue, so we are making the changes here, right?

Reply:
Right, yeah. Nothing in external MRTK asserted that this was happening. `SpatialAwarenessMeshDisplayOptions` is mostly used to determine which material from the MRTK settings to use. This is more an internal implementation detail that is okay to remove going forward.